packages/icons/svg/carAutomatic.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
should this be `fill="currentcolor"`?

Reply:
sorry never mind for here, but the lines below might want to be that rather than the fix text colour

Reply:
discussed in slack and looks like it does work without my suggestion